include/stumpless/facility.h

```diff
@@ -415,6 +418,9 @@ stumpless_get_facility_enum( const char *facility_string );
  * @param facility_string The facility name to get the enum from.
  * 
  * @param facility_buffer_length The length of the buffer
+ * 
+ * @error Clears previous error on success.
+ * @error Raises STUMPLESS_INVALID_FACILITY on invalid input.
```

Comment:
Same here as above, replace the non-existent `@error` commands with a description of the behavior.